src/marten/template/tag/localize.cr

```diff
@@ -0,0 +1,99 @@
+require "./concerns/*"
+
+module Marten
+  module Template
+    module Tag
+      # The `localize` template tag.
+      #
+      # The `localize` template tag allows performing localization on values such as dates, numbers, and times
+      # within templates. It can take one mandatory argument (the value to localize) and an optional `format`
+      # keyword argument to specify a localization format.
+      #
+      # Usage examples:
+      # ```
+      # {% localize created_at %}
+      # {% localize 100000 format: "short" %}
+      # ```
+      #
+      # Optionally, the result of the localization can be assigned to a variable using `as` keyword:
```

Comment:
Can we specify that the specified format must be one of the formats defined in the locale files too? 🙏